SPECS/kernel/kernel.spec

```diff
@@ -194,7 +196,7 @@ fi
 make VERBOSE=1 KBUILD_BUILD_VERSION="1" KBUILD_BUILD_HOST="CBL-Mariner" ARCH=%{arch} %{?_smp_mflags}
 
 # Compile perf, python3-perf
-make -C tools/perf PYTHON=%{python3} all
+make -C tools/perf V=1 NO_PERF_READ_VDSO32=1 NO_PERF_READ_VDSOX32=1 WERROR=0 NO_LIBUNWIND=1 HAVE_CPLUS_DEMANGLE=1 NO_GTK2=1 NO_STRLCPY=1 NO_BIONIC=1 LIBBPF_DYNAMIC=1 LIBTRACEEVENT_DYNAMIC=1 PYTHON=%{python3} all 
```

Comment:
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-3/6.6.35.1/tools/perf/Makefile.config

Reply:
NO_PERF_READ_VDSO32 and NO_PERF_READ_VDSOX32 : disable VDSO (Virtual Dynamically-linked Shared Object) for 32-bit and x32-bit 
NO_LIBUNWIND : no [libunwind](https://github.com/libunwind/libunwind) present
HAVE_CPLUS_DEMANGLE : Indicates that the build should include support for C++ symbol demangling.
NO_GTK2 : disables [GTK2 ](https://gitlab.gnome.org/GNOME/gtk/)support
NO_STRLCPY : Disables the inclusion of strlcpy function
NO_BIONIC : Disables Bionic library
LIBBPF_DYNAMIC ans LIBTRACEEVENT_DYNAMIC :  Enable dynamic linking for libbpf and libtraceevent libraries respectively.